Node API/node-rest-template/test/web/genres.router.spec.js

```diff
@@ -0,0 +1,36 @@
+const request = require('supertest');
+const { initializeApp } = require('../../lib');
+
+const errorCodeToStatusMap = require('../../src/config/errorCodeToStatusMap');
+const { Genre } = require('../../src/lib/genres/genres.model');
+const genresRouter = require('../../src/web/genres.router');
+
+const genres = [{ name: 'action' }, { name: 'drama' }, { name: 'terror' }];
+const serviceResponse = {
+  genres,
+};
+
+describe('genres/router', () => {
+  const app = initializeApp({ router: genresRouter, errorCodeToStatusMap });
+  const server = app.listen();
+
+  afterAll(() => {
+    server.close();
+  });
+
+  describe('GET /', () => {
+    beforeEach(async () => {
+      await Genre.query().del();
+      await Genre.query().insert(genres);
+    });
+
+    it('should return list of genres', async () => {
+      const url = `/`;
```

Comment:
This is not really needed since it's a single value and only used once, simply write the url in the request.